src/figuresClasses.ts

```diff
@@ -1,19 +1,75 @@
 export interface Figure {
-
+  color: string,
+  shape?: string,
```

Comment:
create enum Shape and type Color and use it everywhere
```suggestion
  color: Color,
  shape: Shape,
```

Reply:
what about getArea?